cal.com/app-directory-boilerplate-calcom/index.ts

```diff
@@ -291,12 +291,48 @@ const handleFile: Filemod<
 		const nestedPathWithoutExtension =
 			(parsedPath.dir.split('/pages/')[1] ?? '') + '/' + parsedPath.name;
 
-		const pageContent = `import Page from "@pages/${nestedPathWithoutExtension}";
-import { _generateMetadata } from "app/_utils";
-		
-export const generateMetadata = async () => await _generateMetadata(() => "", () => "");
+		let pageContent = '';
+		if (newPagePath.includes('(individual-page-wrapper')) {
```

Comment:
Can we move this code to the `getPageContent()` fn ?

Reply:
addressed